docs/operations_tasks/Retry-Failed-Bundle-Exporting.md

```diff
@@ -4,65 +4,25 @@ title: Retry failed ingest exporter
 parent: Operations tasks
 ---
 
-# How to retry failed bundles in the Ingest Exporter
+# How to retry failed experiments in the Ingest Exporter
 
-If a submission got stuck in `Processing` state, it's possible that there are some bundles that failed to export.
-Determine the cause of failure. After fixing it, the bundle exporting can be triggered again by resending the exporter messages to RabbitMQ queue
+If a submission got stuck in `Processing` state, it's possible that there are some experiments that failed to export.
+Determine the cause of failure. After fixing it, exporting can be triggered again by redeploying exporter.
```

Comment:
There's a less disruptive way to identify and delete the failed pods. I'll add the details later.

Reply:
@aaclan-ebi in another PR or this one?

Reply:
I can add it and create another PR later.